joomla/joomla-cms
# [4.0] Fix js ajax for pre update checker

### Summary of Changes
Fix js ajax for pre update checker


### Testing Instructions
- Install an extension
- Go to joomla update
[update 07.07.2020]
- Click on `Options` on the right corner
- Change `Update Chanel` to `Custom Url`
- Change `Minimum Stability` to `Development`
- Insert `https://update.joomla.org/core/nightlies/next_major_list.xml` as `Custom URL` than `Save and close`
- Check for updates

### Actual result BEFORE applying this Pull Request
Notice the pre update checker cant find the combat tag
All installed extensions show the flag: Missing Compatibility Tag

![grafik](https://user-images.githubusercontent.com/10517822/86529832-345b2280-beb4-11ea-8c59-81860f818f34.png)


### Expected result AFTER applying this Pull Request
The pre update checker can now find the combat tag (npm required)
All installed extensions without update server show the flag: `Missing Compatibility Tag`
All installed extensions with update server but without update block with a `<targetplatform>` tag for Joomla 4 show the flag: `No`
All installed extensions with update server and an update block with a `<targetplatform>` tag for Joomla 4 show the flag: `Yes`

![grafik](https://user-images.githubusercontent.com/10517822/86529846-52288780-beb4-11ea-9ed3-72165f00cb67.png)
